### PR TITLE
Add documentation explaining how to run the WEP pipeline on the USDF batch system

### PR DESCRIPTION
**Pull request by jbkalmbach**

### CODE REVIEW

File: doc/user-guide/wepBatchProcessing.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+       # Instructions to the butler
+       payload:
+         # This will be the name of the collection after the added prefix of 'u/$USER/'
+         payloadName: latiss_wep_test
+         # This is the location of the butler repository with the data you need
+         butlerConfig: /sdf/group/rubin/repo/embargo/butler.yaml
+         # These are the collections in that repository with the data you want
+         inCollection: LATISS/calib/unbounded,LATISS/raw/all,LATISS/runs/quickLook
+         # Specify a set of images with the query here
+         dataQuery: exposure in (2023020200434, 2023020200435)
```
Comment (jmeyers314) [inline doc/user-guide/wepBatchProcessing.rst:66]: This corresponds to one intra and one extra focal exposure I think, right?  Might want to clarify that.

File: doc/user-guide/wepBatchProcessing.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+    .. code:: bash
+
+        allocateNodes.py -v --dynamic -n 20 -c 32 -m 4-00:00:00 -q roma,milano -g 900 s3df
+
+From the `USDF documentation <https://developer.lsst.io/usdf/batch.html>`_:
+
+    - `s3df` is specified as the target platform.
+    - The `-q roma,milano` option specifies that the glide-in jobs may run in either the roma or milano partition.
+    - The `-n 20 -c 32` options request 20 individual glide-in slots of size 32 cores each (each is a Slurm job that obtains a partial node).
```
Comment (jmeyers314) [inline doc/user-guide/wepBatchProcessing.rst]: This seems way overkill for the single pair of exposures in the example below.  Maybe say something about what reasonable ranges are for a given kind of run?

File: doc/user-guide/wepBatchProcessing.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    - The maximum possible time is set to 4 days via `-m 4-00:00:00`.
+      The glide-in Slurm jobs may not run for the full 4 days however, as the option `-g 900` specifies a condor glide-in shutdown time of 900 seconds or 15 minutes.
+      This means that the htcondor daemons will shut themselves down after 15 minutes of inactivity (for example, after the workflow is complete), and the glide-in Slurm jobs will exit at that time to avoid wasting idle resources.
+    - The `--dynamic` option requests that the htcondor slots be dynamic, partionable slots; this is the recommended setting as it supports possible multi-core jobs in the workflow.
+
+2. Now one can run the `ctrl_bps_htcondor <https://pipelines.lsst.io/modules/lsst.ctrl.bps.htcondor/userguide.html>` workflow
```
Comment (jmeyers314) [inline doc/user-guide/wepBatchProcessing.rst]: I'm not sure this is an actual step, no?  Could just be the first sentence after combining the current steps 2 & 3.

File: doc/user-guide/wepBatchProcessing.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+Additional Notes
+----------------
+
+1. Check on job status with `bps report`
+2. You can overwrite the data query in the batch ``yaml`` file from the command line with ``-d``.
+   For example:
+
+   .. code:: bash
+
+    bps submit -d "exposure.observation_type='cwfs' and exposure.day_obs=20220912 and exposure.seq_num in (96..97)"  bps_wep_test.yaml
```
Comment (jmeyers314) [inline doc/user-guide/wepBatchProcessing.rst:79]: Should add a note about where to expect the output to be available.  I.e., what is the default collection and how to override that if desired.

**Review by jmeyers314 — CHANGES_REQUESTED** [review]
Hi Bryce,
  Sorry this has been so slow.  Looks mostly good to me, I just had a few questions/comments.

File: doc/user-guide/wepBatchProcessing.rst (hunk 2 above)
Comment (jbkalmbach) [reply doc/user-guide/wepBatchProcessing.rst]: Adding suggestions on running with WEP and appropriate `-n` and `-c` values.

File: doc/user-guide/wepBatchProcessing.rst (hunk 4 above)
Comment (jbkalmbach) [reply doc/user-guide/wepBatchProcessing.rst:79]: Added that information and a link to the `bps submit` documentation.

**Review by jmeyers314 — APPROVED** [review]

**Comment by jmeyers314** [comment]
Thanks Bryce.  LGTM!

**Pull request merged**